lib/tests/unit/transforms/test_core.py

```diff
@@ -168,6 +168,8 @@ def test_job_property(thread_local_ret, mocker, monkeypatch):
         mock_func.assert_not_called()
         assert klio_ns._thread_local.klio_job == ret_value
 
+    klio_ns._thread_local.klio_job = None
```

Comment:
This is an odd one.  With beam 2.25, this test suddenly started causing tests in `test_helpers` to start failing.  Apparently the mock stuck around in `_thread_local` and other tests tried to use it 😬